pkg/minikube/kubeconfig/config.go

```diff
@@ -178,3 +182,58 @@ func decode(data []byte) (*api.Config, error) {
 
 	return config.(*api.Config), nil
 }
+
+// GetKubeConfigStatus verifys the ip stored in kubeconfig.
+// If incorrect, the ip is overwritten.
+func GetKubeConfigStatus(ip string, filename string) (string, error) {
```

Comment:
It looks like this function is expected to do two main things: get the old IP out of the kubecfg and compare it to the new IP. It might be a bit cleaner to separate out those two behaviors into `GetIPFromKubeConfig` and then hoist the comparison logic up into a helper function inside the command itself. What do you think?